README.md

```diff
@@ -76,6 +77,32 @@ providing an API to convert user-entered number and date strings - in their
 own format - into actual numbers and dates, and conversely, to format numbers
 and dates into that string format.
 
+Even if the application deals with Enligh loacle only, it may still likely need
+globalization to format programming language bytes into human understandable
+language and vice-versa in a effective and reasonable way.
```

Comment:
If you prefer this, please also add my example. That was heavily inspired by Alex Sexton's intro to messageformat.js. The example makes it much more obvious and uses several aspects of Globalize that are not trivial to implement ad-hoc.